bundles/org.openhab.binding.homematic/README.md

```diff
@@ -14,13 +14,22 @@ When the option "Restricted access" is used, some ports have to be added to the
 2000;
 2001;
 2010;
+8181;
 8701;
 9292;
 ```
 
 Also the IP of the device running openHAB has to be set to the list of "IP addresses for restricted access".
 
-Also under `Home page > Settings > Control panel` with the menu `Security` the option `Authentication` has to be disabled as the binding does not support the configuration of `username` and `password`for the XML-RPC API.
+Also under `Home page > Settings > Control panel` with the menu `Security` the option `Authentication` has to be disabled if the option 'useAuthentication' is not set. This option may to be enabled 
+if the option 'useAuthentication' is set and BIN-RPC is not used. In this case a user and a password must be created. This can be done under `Home page > Settings > Control panel` with the menu `User management`. The new user should have the following configuration:
```

Comment:
Minor but needed,  every sentence should go on its own line. Maybe also have a second look at the grammer.

Reply:
Done